### DISCUSSION TITLE
[Question]: Why does the website refuse to connect?

### DISCUSSION THREAD

**Discussion by 20090601** [danny-avila/LibreChat#770] [Troubleshooting]
### Contact Details

poeusergongxiang@outlook.com

### What is your question?

Now, entering the URL https://inter.klona.ai/chat/new on the webpage is no longer possible. The webpage shows that the site refused to connect, and I don’t know why.

### More Details

No more

### What is the main subject of your question?

Other

### Screenshots

https://s4.aconvert.com/convert/p3r68-cdx67/ari5x-6dcsw.jpg

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It should be possible. Check your NGINX setup.

**Comment by 20090601** [comment]
I have checked it and there is no problem with my NGINX, but I still can't access the website.

**Reply by danny-avila** [reply]
This is something with your configuration and I can't see how you've done it. The app is perfectly accessible via /chat/new on my deployed version, which also uses a domain name
